lib/oga/entity_decoder.rb

```diff
@@ -13,6 +13,7 @@ def self.try_decode(input, html = false)
     # @return [String]
     #
     def self.decode(input, html = false)
+      fail ArgumentError, "decode was sent nil" if input.nil?
```

Comment:
Why is this added? The code in `Oga::EntityDecoder.try_decode` already makes sure this particular `decode` method never receives `nil` as input.